### ISSUE TITLE
Horizontal and vertical stripes

### ISSUE THREAD

**Issue by jnoylinc** [prote376/DBPI-BlindSR#2]
Thanks for your work，I test your code on my own data，there are some Horizontal and vertical stripes
Do you known the reason？

**Comment by prote376** [comment]
Maybe the stripes are caused by jpeg compression artifact. Since the proposed method does not deal with block artifact, it could be a problem if block artifact is included in the input image. To prevent the problem, you could slightly blur input images before applying the code.

**Comment by jnoylinc** [comment]
Thanks for your answer，I also find some block artifact in your results，you think that the artifact comes from jpeg compression artifact？

**Comment by prote376** [comment]
Yes, I think that the problem is caused by jpeg artifact. Could you give me the input image you used and the corresponding result to verify the problem?

**Comment by jnoylinc** [comment]
can you give me your email，I can not find the email in your paper, thanks

**Comment by prote376** [comment]
Please send images to prote376@gmail.com. Thank you.

**Comment by jnoylinc** [comment]
I have sent my data to you，Do you think the block artifact comes from the SR net？

**Comment by jnoylinc** [comment]
Hi, the Horizontal and vertical stripes disappear when I add noise in
g_downup = self.U(torch.clamp(g_pred.detach() + norm_noise, -1, 1))
the result is good! thanks for your work
